.vscode/launch.json

```diff
@@ -64,5 +76,13 @@
 				"start",
 			]
 		},
+	],
+	"inputs": [
+		{
+			"type": "promptString",
+			"id": "functionalTestSuiteMethod",
+			"description": "The method name withing the functional test suite that you wish to debug.",
```

Comment:
just a minor typo here but lgtm otherwise
This should be very similar to the changes described by Chetan on his PR right?

Reply:
it's pulling in exactly Chetan's PR from main. So when we merge versioning-2 into main, the git blame will still show that the change comes from Chetan, and there should be no issue.

Reply:
Gotcha, thanks!